src/main/java/ua/khshanovskyi/controller/NasaController.java

```diff
@@ -0,0 +1,23 @@
+package ua.khshanovskyi.controller;
+
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.RequiredArgsConstructor;
+import ua.khshanovskyi.service.NasaService;
+
+@RestController
+@RequestMapping("/mars/pictures/largest")
```

Comment:
Just a thought: It's better to move '/largest' path part to the corresponding method, to have more flexibility in controller endpoint methods.